app/operator/reference/custom-resources.md

```diff
@@ -3940,7 +3940,7 @@ KonnectGatewayControlPlane is the Schema for the KonnectGatewayControlplanes API
 
 | Field | Description |
 | --- | --- |
-| `apiVersion` _string_ | `konnect.konghq.com/v1alpha1`
+| `apiVersion` _string_ | `konnect.konghq.com/v1alpha2`
```

Comment:
Given that we're updating this file I believe it'd be worth to update the CRD definition doc as well (at the very least I think we should update the `KonnectGatewayControlPlaneSpec` to reflect the changes in the manifests).

Reply:
Do we need to update it manually or is this section generated?

Reply:
Done in: https://github.com/Kong/developer.konghq.com/pull/2657